user/apt-packages.md

```diff
@@ -0,0 +1,19 @@
+---
+title: APT Package Installation
+layout: en
+permalink: /user/apt-packages/
+---
+<div id="toc">
+</div>
+
+When using the [container based infrastructure](/user/workers/container-based-infrastructure/), there is a limitation on
+the use of `sudo` without a password, as is required to install packages with `apt-get`.  In order to have Travis CI do
+this on your behalf prior to `sudo` privileges being disabled, use the `addons.apt_packages` key, e.g.:
```

Comment:
"prior to `sudo` privileges being disabled" → "before `sudo` privilege is removed`